src/.buildkite/publish/docker-compose.yml

```diff
@@ -15,6 +15,7 @@ services:
       - TEST_MYSQL_URI=mysql://root:root@mysql:3306/tests
       - TEST_MARIADB_URI=mysql://root:root@mariadb:3306/tests
       - TEST_MSSQL_URI=sqlserver://SA:Prisma1-prisma@mssql:1433/master
+      - TEST_MSSQL_JDBC_URI=sqlserver://mssql:1433;database=master;user=SA;password=Prisma1-prisma;trustServerCertificate=true;encrypt=DANGER_PLAINTEXT
```

Comment:
This separate thing is needed for now because the SDK utils do not support MSSQL yet (specifically credential transforms & database creation).

So I'm using the official `mssql` library to do this stuff. Unfortunately, it does not support JDBC style connection strings like Prisma does. So there are 2 URLs, one for `mssql`, and one for Prisma.

Once SDK supports MSSQL operations, we can parse the URL, and provide arguments as separate KV pairs, so we won't need an extra URL only for `mssql`